DeviceBridgeE2E/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Development configs
+**/user-config.*
+/cluster-config.json
+/.env
+
+# Node
+**/node_modules/
+**/npm-debug.log*
+
+# Code coverage (nyc)
+/coverage/
+/.nyc_output/
+
+# Build output
+/dist/
+/*.tgz
+
+# Docker output
+/*.tar
+
+out/*
```

Comment:
Looks like this folder was included in your branch before you ignored. You might need to manually remove them from the branch.